pairing/bls12381/circl/scalar.go

```diff
@@ -115,3 +116,11 @@ func (s *Scalar) Pick(stream cipher.Stream) kyber.Scalar {
 }
 
 func (s *Scalar) SetBytes(data []byte) kyber.Scalar { s.inner.SetBytes(data); return s }
+
+func (s *Scalar) ByteOrder() kyber.ByteOrder {
+	return kyber.LittleEndian
+}
```

Comment:
Are you sure?

Looking at
https://github.com/cloudflare/circl/blob/456fe4165f8499b2e6e982891d1a40b8b39cb941/ecc/bls12381/ff/scalar.go#L67-L88
it seems it's marshaling data in Big Endian, no?

Can you add some tests that would validate this?

Reply:
Yes, it seems so, I will be adding tests for it.

Reply:
I added a test for kilic and circl, let me know what you think of it ;)